Multisig/Features/Security/KeychainStorage.swift

```diff
@@ -0,0 +1,170 @@
+//
+//  KeychainCenter.swift
+//  Multisig
+//
+//  Created by Dmitry Bespalov on 02.11.22.
+//  Copyright © 2022 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+import CommonCrypto
+import LocalAuthentication
+
+class KeychainStorage {
+
+    static let sensitivePublicKeyTag = "global.safe.sensitive.public.key"
+    static let sensitivePublicKekTag = "global.safe.sensitive.public.kek"
+    static let sensitiveEncryptedPrivateKeyTag = "global.safe.sensitive.private.key.as.encrypted.data"
+    static let derivedPasswordTag = "global.safe.password.as.data"
+    static let sensitiveKekTag = "global.safe.sensitive.KEK"
```

Comment:
is this a sensitive private kek tag? Maybe good to make it consistent with the public key variable